tests/phpunit/tests/blocks/template-part.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Tests for template part rendering.
+ *
+ * @package WordPress
+ *
+ * @group block-templates
+ */
+class Tests_Block_Template_Part extends WP_UnitTestCase {
+	/**
+	 * @ticket 59318
+	 */
+	public function test_render_block_core_template_part_missing_template() {
```

Comment:
```suggestion
	public function test_returns_empty_string_when_template_is_missing() {
```

Question: would it be better to rename test to describe what it is testing?

Also should we add test coverage for when debug is truthy to assert that one of the following happens (depending on what is missing - see code):

- empty string (placeholder) 
- error message